nix/overlay.nix

```diff
@@ -53,6 +53,11 @@ let
         # Use base package set from pyproject.nix builders
         (pkgs.callPackage pyproject-nix.build.packages {
           inherit python;
+          stdenv = pkgs.stdenv.override {
+            targetPlatform = pkgs.stdenv.targetPlatform // {
+              darwinSdkVersion = "12.0";
```

Comment:
This isn't strictly required for python 3.13 support, but it sure does make life easier by allowing more wheels to be used on macOS. A bunch of the more-complex-to-build wheels like scipy and scikit-learn are only available using a 12.0 darwin SDK build.